src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/Interfaces/IVsTestConsoleWrapperAsync.cs

```diff
@@ -82,18 +78,13 @@ public interface IVsTestConsoleWrapperAsync
         Task RunTestsWithCustomTestHostAsync(IEnumerable<TestCase> testCases, string runSettings, TestPlatformOptions options, ITestRunEventsHandler testRunEventsHandler, ITestHostLauncher customTestHostLauncher);
 
         /// <summary>
-        /// See <see cref="IVsTestConsoleWrapper.CancelTestRun"/>.
-        /// </summary>
-        void CancelTestRun();
```

Comment:
>CancelTestRun [](start = 13, length = 13)

Is there any way we could keep this by hiding it in the derived interface?

Reply:
Removing APIs would definitely break people.

---
In reply to: [448171317](https://github.com/microsoft/vstest/pull/2463#discussion_r448171317) [](ancestors = 448171317)

Reply:
fixed